assets/prototype/application/services/entityTextFieldSearch/index.ts

```diff
@@ -1,3 +1,8 @@
+/**
```

Comment:
Changes from this file were pulled into a related PR: https://github.com/eventespresso/event-espresso-core/pull/2157